runner/runner.go

```diff
@@ -35,7 +36,12 @@ func Run(settings *options.Settings) {
 	ht.UserAgent = fmt.Sprintf("%v/%s", os.Args[0], version)
 	setupMetrics(settings)
 
-	ph := tokeninfoproxy.NewTokenInfoProxyHandler(settings.UpstreamTokenInfoURL, settings.UpstreamCacheMaxSize, settings.UpstreamCacheTTL)
+	var ph http.Handler
+	if settings.UpstreamTokenInfoURL != nil {
+		ph = tokeninfoproxy.NewTokenInfoProxyHandler(settings.UpstreamTokenInfoURL, settings.UpstreamCacheMaxSize, settings.UpstreamCacheTTL)
+	} else {
+		ph = errorall.NewErrorAllHandler()
```

Comment:
@vetinari why not setting settings.UpstreamTokenInfoURL := errorall.NewErrorAllHandler by default, and leave the original code?